internal/pkg/log/logger.go

```diff
@@ -52,6 +54,86 @@ func (l *zapLogger) Log(level string, args ...any) {
 	}
 }
 
+func getMessage(args []any) string {
+	if len(args) == 1 {
+		if str, ok := args[0].(string); ok {
+			return str
+		}
+	}
+	return fmt.Sprint(args...)
+}
+
+func (l *zapLogger) LogCtx(ctx context.Context, level string, args ...any) {
+	message := getMessage(args)
+	fields := make([]zap.Field, 0)
+	switch level {
+	case "debug", "DEBUG":
+		l.Desugar().Debug(message, fields...)
```

Comment:
There is no way to pass both Sprintf args and fields into SugaredLogger so I'll need to use this lower level API.